public/class-whmcs-wordpress-public.php

```diff
@@ -0,0 +1,141 @@
+    private $plugin_name;
+
+    /**
+     * The version of this plugin.
+     *
+     * @since    1.0.0
+     * @access   private
+     * @var      string    $version    The current version of this plugin.
+     */
+    private $version;
+
+    /**
+     * Initialize the class and set its properties.
+     *
+     * @since    1.0.0
+     * @param      string    $plugin_name       The name of the plugin.
+     * @param      string    $version    The version of this plugin.
+     */
+    public function __construct($plugin_name, $version) {
+        $this->plugin_name = $plugin_name;
+        $this->version = $version;
+    }
+
+    /**
+     * Register the stylesheets for the public-facing side of the site.
+     *
+     * @since    1.0.0
+     */
+    public function enqueue_styles() {
+
+        /**
+         * This function is provided for demonstration purposes only.
+         *
+         * An instance of this class should be passed to the run() function
+         * defined in Whmcs_Wordpress_Loader as all of the hooks are defined
+         * in that particular class.
+         *
+         * The Whmcs_Wordpress_Loader will then create the relationship
+         * between the defined hooks and the functions defined in this
+         * class.
+         */
+
+        wp_enqueue_style($this->plugin_name, plugin_dir_url(__FILE__) . 'css/whmcs-wordpress-public.css', [], $this->version, 'all');
+    }
+
+    /**
+     * Register the JavaScript for the public-facing side of the site.
+     *
+     * @since    1.0.0
+     */
+    public function enqueue_scripts() {
+
+        /**
+         * This function is provided for demonstration purposes only.
+         *
+         * An instance of this class should be passed to the run() function
+         * defined in Whmcs_Wordpress_Loader as all of the hooks are defined
+         * in that particular class.
+         *
+         * The Whmcs_Wordpress_Loader will then create the relationship
+         * between the defined hooks and the functions defined in this
+         * class.
+         */
+
+        wp_enqueue_script($this->plugin_name, plugin_dir_url(__FILE__) . 'js/whmcs-wordpress-public.js', ['jquery'], $this->version, false);
+    }
+
+    public function shortcode_whmcs_wordpress($atts = [], $content = null) {
+        $buildDir = __DIR__ . '/form';
+
+        $cssFiles = scandir("$buildDir/css");
+        $jsFiles = scandir("$buildDir/js");
+        $fontFiles = scandir("$buildDir/fonts");
+        $buildFiles = array_merge($cssFiles, $jsFiles, $fontFiles);
+
+        $files = array_filter($buildFiles, function ($file) {
+            return $file !== '.' && $file !== '..';
+        });
+
+        array_map(function ($file) {
+            $posDotBeforeExt = strripos($file, '.');
+            $fileExt = substr($file, $posDotBeforeExt + 1);
+
+            switch ($fileExt) {
+                case 'js':
+                    wp_enqueue_script(uniqid(), plugin_dir_url(__FILE__) . "form/js/$file");
```

Comment:
@brunoferreiralkn Por gentileza, poderia verificar se o css/js será carregado apenas na página que o shortcode for inserido?